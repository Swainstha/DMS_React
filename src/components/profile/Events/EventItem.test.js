import testSnapFunction from '../../../util/testSnapFunction';
import EventItem from './EventItem'
import React from 'react';
const dummy_data= {
   data:{
       title:"title",
      image:"image.png",
      address:"patan",
      date:"Nov 3, 2020"
   }
}

testSnapFunction("<EventItem />", "Snapshot test for EventItem ",<EventItem {...dummy_data}/> )