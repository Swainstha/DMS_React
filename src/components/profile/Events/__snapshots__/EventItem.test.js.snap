// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EventItem /> Snapshot test for EventItem  1`] = `ShallowWrapper {}`;
